backend/coins/btc/transaction.go

```diff
@@ -60,7 +60,10 @@ func (account *Account) getFeePerKb(args *accounts.TxProposalArgs) (btcutil.Amou
 	}
 	var feeTarget *FeeTarget
 	for _, target := range account.feeTargets() {
-		if target.code == args.FeeTargetCode {
+		if len(args.PaymentRequests) > 0 {
+			// for payment requests we just take the last element of the array, which is the highest fee target
+			feeTarget = target
```

Comment:
This is a bit fragile as it relies on implicit assumptions. It would be better to make `type FeeTargets []*FeeTarget`, then add a `highest()` method on it so it is explicit.

frontend still has `feeTarget: 'high',` which is confusing. Could it be removed, or if not, add a comment that the presence of a paymentRequest overrides it.

Reply:
Great suggestion! Done :+1: 

I also changed the feeTarget in the FE from `high` to `custom` and added a comment to clarify :pray: